RELEASING.md

```diff
@@ -4,7 +4,6 @@ Release Guide
 1. Bump the version number (`sed -i -e 's/1.2.1/1.2.2/g' *.te`).
 1. Create a PR with all the changes above and merge it after a review.
 1. Create a release on github. It will create the corresponding tag.
-1. In that release, copy and paste the git commit titles since the last release
-(`git shortlog 1.2.1...master --no-merges | grep -E "^(\s)+\w" | sed -e 's/     /*/'`)
+1. In that release, Select "Auto-generate release notes"
```

Comment:
How does that work?

Reply:
@ekohl Here's what it looks like: https://github.com/pulp/pulpcore-selinux/releases/tag/1.2.8

Reply:
That doesn't really answer my question :) Those are the results but I'm wondering how it works and if it makes sense (for me) to implement in other repositories.

For example, does it handle multiple version streams? We have a -stable branch and I don't want to get confused if I release (in order) 1.0.0, 1.1.0, 1.0.1. It may use git to do the right thing, but I'm interested because I've also seen other tools (https://github.com/github-changelog-generator/github-changelog-generator) do the wrong thing.

Reply:
Good question. I guess the way I could test that is to create a new branch like "1.2.7", and go to create a 1.2.7.1 tag.

Also, I plan on using this feature only for this repository because it lacks our usual changelog automation.